.gitignore

```diff
@@ -21,4 +21,6 @@ npm-debug.log
 /rt.sh
 /grt.sh
 tooling/release.json
-report.xml
\ No newline at end of file
+report.xml
+/test/source/core/*
+/test/source/core
```

Comment:
I think `/test/source/core/*` can be removed, as `/test/source/core` ignores full folder's content